Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -441,6 +441,7 @@ void Adjuster::adjust(RenderStyle& style, const RenderStyle* userAgentAppearance
         // Framesets ignore display and position properties.
         style.setPosition(PositionType::Static);
         style.setEffectiveDisplay(DisplayType::Block);
+        style.setFloating(Float::None);
```

Comment:
Please update the commit message to reflect what this PR does and also update the comment at line#441.

Reply:
@alanbaradlay - Took some time to get back to this but updated commit message (removed fluff) and also updated inline comment as well.